spec/cucumber/rake/task_spec.rb

```diff
@@ -7,6 +7,15 @@
 module Cucumber
   module Rake
     describe Task do
+      describe '@task_name' do
+        context 'has read access to task name' do
+          it { expect(subject.respond_to? :task_name).to be true }
```

Comment:
this can be slightly simplified to `expect(subject).to respond_to(:task_name)` And the one below can be `.not_to`